diff --git a/node_modules/@types/react-native-vector-icons/Icon.d.ts b/node_modules/@types/react-native-vector-icons/Icon.d.ts
index 455a326..dfe68bc 100644
--- a/node_modules/@types/react-native-vector-icons/Icon.d.ts
+++ b/node_modules/@types/react-native-vector-icons/Icon.d.ts
@@ -6,7 +6,7 @@ import {
   TouchableHighlightProps,
   TouchableNativeFeedbackProps,
   TabBarIOSItemProps,
-  ToolbarAndroidProps
+  ToolbarAndroidProps as ToolbarAndroidProps2
 } from 'react-native';
 
 export interface IconProps extends TextProps {
@@ -74,7 +74,7 @@ export interface IconButtonProps extends IconProps, TouchableHighlightProps, Tou
 
 export type ImageSource = any;
 
-export interface ToolbarAndroidProps extends ToolbarAndroidProps {
+export interface ToolbarAndroidProps extends ToolbarAndroidProps2 {
   /**
    * Name of the navigation logo icon
    * (similar to ToolbarAndroid logo)
