patch-package
--- a/node_modules/next/dist/server/document.js
+++ b/node_modules/next/dist/server/document.js
@@ -190,7 +190,7 @@ function (_Component2) {
         });
       }
 
-      return _react.default.createElement("head", this.props, head, page !== '/_error' && _react.default.createElement("link", {
+      return _react.default.createElement("head", this.props, children, head, page !== '/_error' && _react.default.createElement("link", {
         rel: "preload",
         href: "".concat(assetPrefix, "/_next/static/").concat(buildId, "/pages").concat(pagePathname),
         as: "script",
@@ -205,7 +205,7 @@ function (_Component2) {
         href: "".concat(assetPrefix, "/_next/static/").concat(buildId, "/pages/_error.js"),
         as: "script",
         nonce: this.props.nonce
-      }), this.getPreloadDynamicChunks(), this.getPreloadMainLinks(), this.getCssLinks(), styles || null, children);
+      }), this.getPreloadDynamicChunks(), this.getPreloadMainLinks(), this.getCssLinks(), styles || null);
     }
   }]);
   return Head;
