diff --git a/node_modules/react-native-rate/index.d.ts b/node_modules/react-native-rate/index.d.ts
index f657d66..04e8e6e 100644
--- a/node_modules/react-native-rate/index.d.ts
+++ b/node_modules/react-native-rate/index.d.ts
@@ -6,7 +6,6 @@ declare module 'react-native-rate' {
 		OtherAndroidURL?: string,
 		preferredAndroidMarket?: AndroidMarket,
 		preferInApp?: boolean,
-		openAppStoreIfInAppFails?: boolean,
 		inAppDelay?: number,
 		fallbackPlatformURL?: string,
 		openAppStoreIfInAppFails?: boolean,
