patch-package
--- a/node_modules/react-native-linear-gradient/index.d.ts
+++ b/node_modules/react-native-linear-gradient/index.d.ts
@@ -1,9 +1,9 @@
 import * as React from "react";
-import * as ReactNative from "react-native";
+import { ViewProps } from "react-native";
 
 declare module "react-native-linear-gradient" {
 
-    interface LinearGradientProps extends ReactNative.ViewProperties {
+    interface LinearGradientProps extends Pick<ViewProps, Exclude<keyof ViewProps, "start" | "end">> {
         colors: string[],
         start?: { x: number, y: number },
         end?: { x: number, y: number },
