patch-package
--- a/node_modules/react-navigation/src/views/StackView/StackViewLayout.js
+++ b/node_modules/react-navigation/src/views/StackView/StackViewLayout.js
@@ -23,6 +23,8 @@ import { NavigationProvider } from '../NavigationContext';
 import TransitionConfigs from './StackViewTransitionConfigs';
 import { supportsImprovedSpringAnimation } from '../../utils/ReactNativeFeatures';
 
+import ViewOverflow from 'react-native-view-overflow';
+
 const emptyFunction = () => {};
 
 const { width: WINDOW_WIDTH, height: WINDOW_HEIGHT } = Dimensions.get('window');
@@ -441,9 +443,9 @@ class StackViewLayout extends React.Component {
     if (headerMode === 'float') {
       const { scene } = this.props.transitionProps;
       floatingHeader = (
-        <View pointerEvents="box-none" onLayout={this._onFloatingHeaderLayout}>
+        <ViewOverflow pointerEvents="box-none" onLayout={this._onFloatingHeaderLayout}>
           {this._renderHeader(scene, headerMode)}
-        </View>
+        </ViewOverflow>
       );
     }
     const {
