patch-package
--- a/node_modules/react-native-macos/Libraries/Components/ScrollView/ScrollView.js
+++ b/node_modules/react-native-macos/Libraries/Components/ScrollView/ScrollView.js
@@ -679,7 +679,7 @@ const ScrollView = createReactClass({
       ScrollContentContainerViewClass = View;
     } else if (Platform.OS === 'macos') {
       ScrollViewClass = RCTScrollView;
-      ScrollContentContainerViewClass = View;
+      ScrollContentContainerViewClass = this.props.flipped ? RCTScrollContentView : View;
     }
 
     invariant(
@@ -911,6 +911,7 @@ if (Platform.OS === 'android') {
     {
     },
   );
+  RCTScrollContentView = requireNativeComponent('RCTScrollContentView', View);
 }
 
 module.exports = ScrollView;
--- a/node_modules/react-native-macos/Libraries/Components/Touchable/TouchableOpacity.js
+++ b/node_modules/react-native-macos/Libraries/Components/Touchable/TouchableOpacity.js
@@ -179,6 +179,8 @@ var TouchableOpacity = createReactClass({
    * defined on your component.
    */
   touchableHandleActivePressIn: function(e: Event) {
+    console.log('touchableHandleActivePressIn');
+    console.log(e);
     if (e.dispatchConfig.registrationName === 'onResponderGrant') {
       this._opacityActive(0);
     } else {
@@ -188,6 +190,8 @@ var TouchableOpacity = createReactClass({
   },
 
   touchableHandleActivePressOut: function(e: Event) {
+    console.log('touchableHandleActivePressOut');
+    console.log(e);
     this._opacityInactive(250);
     this.props.onPressOut && this.props.onPressOut(e);
   },
--- a/node_modules/react-native-macos/Libraries/Text/TextInputLegacy/RCTTextField.m
+++ b/node_modules/react-native-macos/Libraries/Text/TextInputLegacy/RCTTextField.m
@@ -35,7 +35,8 @@
     self.delegate = self;
     self.drawsBackground = NO;
     self.bordered = NO;
-    self.bezeled = YES;
+    self.bezeled = NO;
+    self.focusRingType = NSFocusRingTypeNone;
     // self.autoresizingMask = NSViewWidthSizable | NSViewHeightSizable;
 
     _eventDispatcher = eventDispatcher;
--- a/node_modules/react-native-macos/React/Modules/RCTAppearance.m
+++ b/node_modules/react-native-macos/React/Modules/RCTAppearance.m
@@ -20,7 +20,7 @@ RCT_EXPORT_MODULE()
                       ofObject:(__unused id)object
                         change:(__unused NSDictionary *)change
                        context:(__unused void *)context {
-  [NSAppearance setCurrentAppearance:NSApp.effectiveAppearance];
+  [NSAppearance setCurrentAppearance:[NSApp mainWindow].effectiveAppearance];
   [self sendEventWithName:@"onAppearanceChange" body:[self resolveConstants]];
 }
 
--- a/node_modules/react-native-macos/React/Views/ScrollView/RCTScrollContentView.m
+++ b/node_modules/react-native-macos/React/Views/ScrollView/RCTScrollContentView.m
@@ -16,20 +16,9 @@
 
 @implementation RCTScrollContentView
 
-- (void)reactSetFrame:(CGRect)frame
+- (BOOL)isFlipped
 {
-  [super reactSetFrame:frame];
-
-  RCTNativeScrollView *scrollView = (RCTNativeScrollView *)self.superview.superview;
-
-  if (!scrollView) {
-    return;
-  }
-
-  RCTAssert([scrollView isKindOfClass:[RCTNativeScrollView class]],
-            @"Unexpected view hierarchy of RCTScrollView component.");
-
-  // [scrollView updateContentOffsetIfNeeded];
+  return NO;
 }
 
 @end
